engine/Drawable.h

```diff
@@ -0,0 +1,19 @@
+#ifndef DRAWABLE_H
+#define DRAWABLE_H
+
+#include "Engine.h"
+
+class Drawable
```

Comment:
please rename class.
`Drawable` seems to be a member of `X11` library, which is necessary to build the project under **UNIX**